src/lib.rs

```diff
@@ -565,7 +584,20 @@ pub async fn transfer_assets(transfers: TransfersRequest) -> Result<TransfersSer
     if status != 200 {
         return Err(anyhow!("Error calling {endpoint}"));
     }
-    Ok(serde_json::from_str(&transfer_res)?)
+
+    let transfer_res: TransfersSerializeResponse = serde_json::from_str(&transfer_res)?;
+
+    let _declare = async {
```

Comment:
Just out of curiosity, wouldn't be better to remove the `_declare` var? like this:

```rust
    async {
        let declare_endpoint = &get_endpoint("declare").await;
        let (_declare_res, status) = post_json(declare_endpoint, &transfer_res.declare).await?;
        if status != 200 {
            return Err(anyhow!("Error calling {declare_endpoint}"));
        }
        Ok(status)
    }
    .await?;
```

Reply:
Yeah